active_learning_multiannotator/active_learning.ipynb

```diff
@@ -374,6 +375,7 @@
     "\n",
     "    # format the data for the next round of active learning, ie. moving some unlabeled \n",
     "    # examples to the labeled pool because we are collecting labels for them\n",
+    "    # note: this is a custom helper function, replace with your own implementation in your application\n",
```

Comment:
```suggestion
    "    # Format the data for the next round of active learning, moving some unlabeled \n",
    "    # examples to the labeled pool because we are collecting labels for them.\n",
    "    # Note: you need to replace this helper function with your own function to properly update the labeled and unlabeled pools in your application\n",
```